server.js

```diff
@@ -7888,6 +7888,73 @@ camp.route(/^\/vaadin-directory\/(star|status|rating|rc|rating-count|v|version|r
 
 }));
 
+// Bugzilla bug integration
+camp.route(/^\/bugzilla\/(\d+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+  var bugNumber = match[1];  // eg, 1436739
+  var format = match[2];
+  var options = {
+    method: 'GET',
+    json: true,
+    uri: 'https://bugzilla.mozilla.org/rest/bug/' + bugNumber
+  };
+  var badgeData = getBadgeData('Bug ' + bugNumber, data);
+  request(options, function (err, res, json) {
+    if (checkErrorResponse(badgeData, err, res)) {
+      badgeData.text[1] = 'unknown';
```

Comment:
could you remove this line, as [`checkErrorResponse()`](https://github.com/badges/shields/blob/3cb71037fdbed20f74a18880ed9edad4273a08c5/lib/error-helper.js#L3) should set `badgeData.text[1]` to a relevant message.